packages/commonwealth/client/scripts/views/components/component_kit/cw_component_showcase.tsx

```diff
@@ -11,6 +11,7 @@ import { CWBreadcrumbs } from './cw_breadcrumbs';
 import { CWButton } from './new_designs/cw_button';
 import { CWUpvote } from './new_designs/cw_upvote';
 import { CWThreadAction } from './new_designs/cw_thread_action';
+import { CWSearchBar } from './new_designs/cw_search_bar';
```

Comment:
we try to ditch `snake_case` for file names. Please use `CWSearchBar`

Reply:
Addressed.